### PR TITLE
Allow cert callback to override skipAddCA

### PR DESCRIPTION
**Pull request by julek-wolfssl**
Addresses ZD16000

### CODE REVIEW

**Comment by julek-wolfssl** [comment]
Issue confirmed fixed by customer.

**Review by JacobBarthelmeh — APPROVED** [review]
Looks good. I'd like to get another pair of eyes on it before merged though.

**Review by dgarske — APPROVED** [review]

**Pull request merged**
